src/index.ts

```diff
@@ -224,6 +249,37 @@ export class AthenaNodeClient {
         return query.results;
     }
 
+
+    /**
+     * Get statistics from a query execution
+     *
+     * @private
+     * @template T
+     *
+     *
+     * @returns {Promise<T[]>} - parsed query result rows
+     * @memberof AthenaNodeClient
+     * @param {string} executionId
+     */
+    private async getQueryStatistics(executionId: string): Promise<Statistics> {
+        const input: GetQueryExecutionCommand = new GetQueryExecutionCommand({
+            QueryExecutionId: executionId
+        });
+
+        const response = await this._client.send(input);
+        const bytes = response?.QueryExecution?.Statistics?.DataScannedInBytes ?? null;
+        const timeInSeconds = response?.QueryExecution?.Statistics?.EngineExecutionTimeInMillis ?? 0 / 1000
```

Comment:
Esto intuyo que lo querías hacer que si la variable esa está vacía, se ponga a `0`, pero tal y cómo lo tenías, se pone valor por defecto `0 / 1000` que es `0` también, pero se entiende peor.

```suggestion
        const timeInSeconds = (response?.QueryExecution?.Statistics?.EngineExecutionTimeInMillis ?? 0) / 1000
```